content/spin/v2/install.md

```diff
@@ -135,7 +135,7 @@ To install a specific version, you can pass arguments to the install script this
 <!-- @selectiveCpy -->
 
 ```bash
-$ curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash -s -- -v v1.5.0
+$ curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash -s -- -v v2.7.0
```

Comment:
Oh wait... I just noticed the context for this is "to install a specific version".  There's no need to keep it revved to latest version because that's the one they get by default.  I wonder if it would be better to make this an obviously (cough) fake version like 99.99.99 or 12.34.5?